makefiles/pseudomodules.inc.mk

```diff
@@ -89,6 +89,7 @@ PSEUDOMODULES += netdev_layer
 PSEUDOMODULES += netdev_register
 PSEUDOMODULES += netstats
 PSEUDOMODULES += netstats_l2
+PSEUDOMODULES += netstats_neighbor_%
```

Comment:
I know, there is already precedence in this file, but let's not make things worse: could you please explicitly name the pseudomodules instead of enforcing all possible sub-modules of `netstats_neighbor_%` to be pseudomodules?